lib/rock_rms/resources/payment_detail.rb

```diff
@@ -0,0 +1,61 @@
+module RockRMS
+  class Client
+    module PaymentDetail
+      def list_payment_details(options = {})
+        options = {'$top' => 10}
+        res = get(payment_detail_path, options)
+        Response::PaymentDetail.format(res)
+      end
+
+      def create_payment_detail(payment_type:, foreign_key: nil, card_type: nil)
```

Comment:
Add support for `AccountNumberMasked` please :)

Reply:
I'm trying to add AccountNumberMasked, but the api seems to be ignoring the field. And in the API docs, I can only find this as a read-only field when you $expand FinancialTransactionDetail on FinancialTransaction

do you know anything more than this?